src/core.ts

```diff
@@ -202,6 +202,16 @@ interface SiteAdapterOptions {
       (Enables special handling of certain HTML elements, which doesn't
        work well on all sites) */
   iframe?: boolean;
+
+  /**
+   * Styling function for the selected row
```

Comment:
Could you make this a tad more specific so it comes out better in the docs? eg "Applies custom styles to the selected row in the original page."